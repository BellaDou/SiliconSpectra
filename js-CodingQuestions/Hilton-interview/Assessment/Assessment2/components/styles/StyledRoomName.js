import styled from "styled-components";

const StyledRoomName = styled.h4`
  font-size: 1.1em;
  text-align: center;
  margin: 0.2em 0;
`;

export default StyledRoomName;
