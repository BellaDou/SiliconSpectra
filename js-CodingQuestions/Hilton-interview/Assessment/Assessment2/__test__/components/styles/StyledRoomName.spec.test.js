import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import StyledRoomName from "../../../components/styles/StyledRoomName";

test("Should properly render style component", () => {
  const tree = renderer.create(<StyledRoomName />).toJSON();
  expect(tree).toMatchSnapshot();
});
