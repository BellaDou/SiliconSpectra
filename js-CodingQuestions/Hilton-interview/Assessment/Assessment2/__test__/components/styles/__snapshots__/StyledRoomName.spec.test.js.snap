// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should properly render style component 1`] = `
.c0 {
  font-size: 1.1em;
  text-align: center;
  margin: 0.2em 0;
}

<h4
  className="c0"
/>
`;
